### DISCUSSION TITLE
What should I do when its saying Running Conflicted?

### DISCUSSION THREAD

**Discussion by Vyeche** [vrtmrz/obsidian-livesync#128] [Q&A]
Hi,

What should I do when it's saying Running Conflicted?

![image](https://user-images.githubusercontent.com/972377/194883064-509e3b34-208f-4e7d-871f-d1362ecfe5ff.png)
![image](https://user-images.githubusercontent.com/972377/194883296-10f7779d-b317-49fa-9ac9-4b1a2f7977e5.png)

**Comment by vrtmrz** [comment]
Reproduced. But I think this might just be a lack of status updates. (Actually, it has been done.)
It is harmless but will be fixed. Thank you for pointing it out!

**Reply by douglasg14b** [reply]
Mine says this as well, and there are files missing that are available on every other device....

One device just will not pull a set of dirs and file changes.
